console-subscriber/src/builder.rs

```diff
@@ -101,4 +104,41 @@ impl Builder {
     pub fn build(self) -> (TasksLayer, Server) {
         TasksLayer::build(self)
     }
+
+    /// Configures this builder from a standard set of environment variables:
+    ///
+    /// * `TOKIO_CONSOLE_RETENTION`: The number of seconds to accumulate
+    ///   completed tracing data. Default: 60s.
+    /// * `TOKIO_CONSOLE_BIND`: a HOST:PORT description, such as
+    ///   localhost:1234 or similar. Default: 127.0.0.1:6669
+    /// * `TOKIO_CONSOLE_PUBLISH_INTERVAL`: The number of milliseconds to wait
+    ///   between sending updates to the console. Default: 1000ms (1s)
+
+    pub fn from_default_env(mut self) -> Self {
```

Comment:
nit: in general, we don't put whitespace between doc comments and functions elsewhere in the project
```suggestion
    ///   between sending updates to the console. Default: 1000ms (1s)
    pub fn from_default_env(mut self) -> Self {
```